tests/api/reports-downloads.php

```diff
@@ -49,8 +49,8 @@ public function test_get_report() {
 
 		// Populate all of the data.
 		$prod_download = new WC_Product_Download();
-		$prod_download->set_file( plugin_dir_url( __FILE__ ) . '/assets/images/help.png' );
-		$prod_download->set_id( 1 );
+		$prod_download->set_file( plugin_dir_url( __FILE__ ) . 'assets/images/help.png' );
+		$prod_download->set_id( '1' );
```

Comment:
Looks like the other calls to `WC_Product_Download::set_id` were missed? (I still see non-strings..)

Reply:
@jeffstieler Thanks, good catch. Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2519/commits/ffd87a0459698f6dd5ce051c1aed122af92a3afb .